test/integration/targets/includes_race/aliases

```diff
@@ -0,0 +1,3 @@
+shippable/posix/group5
+includes_race
```

Comment:
This is redundant and should be removed:

```suggestion
```

Reply:
done